src/components/application_manager/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -72,6 +72,32 @@ mobile_apis::AppHMIType::eType StringToAppHMIType(const std::string &str) {
   }
 }
 
+std::string AppHMITypeToString(mobile_apis::AppHMIType::eType type) {
+  if (mobile_apis::AppHMIType::DEFAULT == type) {
```

Comment:
I could do so, but there is a reverse function (StringToAppHMIType), already in the file which also uses if statements [here](https://github.com/smartdevicelink/sdl_core/blob/master/src/components/application_manager/src/commands/mobile/register_app_interface_request.cc#L49-73). 

Do you just want me to change both of them to switch statements?